### ISSUE TITLE
What do we classify as an "InfoSec" meetup for the purposes of this list?

### ISSUE THREAD

**Issue by hellodanielting** [Infosec-Community/APAC-Meetups#8]
The obviously InfoSec meetups, are SecTalks and Ruxmon.
However, there are some InfoSec adjacent ( eg. Crypto Aus, Digital Privacy), and regularly discusses infosec topics despite not being explicitly InfoSec ( eg. Linux Users Victoria).

Do we include those? If so, do we attempt to categorise, the meetups? Eg. Technical, General, etc...
Thoughts @codingo @hakluke ?

**Comment by codingo** [comment]
I've been thinking about this for the conferences repository for a while. Unfortunately, at some point, our opinion comes into it.

The challenge I see is that I personally don't see a conferences like AustCert to be a technical conference, but a lot of people in different disciplines do. The same challenge applies for AISA meetups. I think it needs to be done, but carefully so not to create the wrong impressions.

Thoughts on solving this? It's bugged me for quite a while.

**Comment by hellodanielting** [comment]
I'm leaning towards, let's include them all, and the categorise them in some fashion that is as objective as possible. There is a lot of knowledge out there. I'd rather be as inclusive as possible.

I guess clearly, something like say, WordPress User Group won't be included, but Privacy Meetup will. Then there are InfoSec support groups like AWSN. I think we should include them.

Will give it some more thought. But looks like this needs more discussion from the community on this.
It will definitely wander into, "why are we doing this in the first place?" question territory.

**Comment by hakluke** [comment]
It would be really great to find an objective way to distinguish the difference between an "infosec" meetup and a "non-infosec" meetup. Personally I don't think meetups such as generic Linux user groups should be on the list as it will create too much noise.

Some infosec cons seem more based around defense or compliance, while others are more about the offensive side of things. Maybe we could even split it up that way? I'm sure there will be some which sit in the middle though.

**Comment by 0x90n** [comment]
I think any meetup with security as the focus - a generic linux group probably isn't the best target there, and if someones keen on that area they'll be able to locate it themselves. Simple threshold; are almost all their meetup events targeted towards *something* security? Happy to categorise as RED/BLUE/other.

In terms of University groups - absolutely do not include them in meetups; some of these are specific for the students only. Rather, let's keep these for the University and Clubs page.

**Comment by codingo** [comment]
Agreed, this thread was a long time before we had  those other repositories - that's definitely the best place for university meetups now that it's established.
